### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Refactor Detail view with grid.

## Screenshots

image

image

## Trello Card

https://trello.com/c/zdbac445/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: maquetado/detail.html [hunk 1]
```diff
@@ -11,23 +11,23 @@
...
+      <img class='book-image' src='assets/book-cover.png'>
+      <img class='book-badge' src='assets/badge.png'>
+    </div>
+    <div class='book-title'>
+      <h1 class='title'>Título del Libro</h1>
+    </div>
+    <div class='title-genre'>
+      <p class='title-genre'>(género)</pclass='title-genre'>
+    </div>
+    <hr class='book-separator'>
```
Comment (erikankodelia) [inline maquetado/detail.html]: It is not necessary, you can move the book-separator attributes to `p` for example

File: maquetado/scss/details.scss [hunk 2]
```diff
@@ -98,27 +110,79 @@
 
 @media only screen and (max-width: 1024px) {
   .book-card {
-    flex-flow: column;
+    box-shadow: 4px 4px 10px $black-russian;
+    display: grid;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Remember that it is not necessary to repeat the same attributes that appear in desktop. Check these classes

File: maquetado/detail.html [hunk 3]
```diff
@@ -17,17 +17,17 @@
...
-      <div class='book-details'>
-        <p class='book-item'><strong>Autor del Libro: </strong> <span class='book-item-span'>del autor del Libro</span></p>
-        <p class='book-item'><strong>Editorial: </strong><span class='book-item-span'>Nombre de editorial</span></p>
-        <p class='book-item'><strong>Año de Publicación: </strong><span class='book-item-span'>Año de Publicación</span></p>
-      </div>
+    <div class='book-title'>
+      <h1 class='title'>Título del Libro</h1>
+    </div>
+    <div class='title-genre'>
+      <p class='title-genre'>(género)</pclass='title-genre'>
```
[the remark this answers is not in the document]
Comment (mcolombo87) [reply maquetado/detail.html]: I don't know how that it's there :eyes:

File: maquetado/scss/details.scss [hunk 4]
```diff
@@ -1,59 +1,67 @@
...
     .book-title {
-      .title-genre {
-        color: $grey;
-        font-size: $font-large;
-        font-weight: bold;
-      }
+      grid-column: 1 / 3;
+      grid-row: 1 / 1;
+      justify-self: center;
+      padding: 0 8px 12px 0;
```
[the remark this answers is not in the document]
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done!

File: maquetado/detail.html (hunk 1 above)
Comment (mcolombo87) [reply maquetado/detail.html]: Oka! I moved it to book-detail and delete that separator.

File: maquetado/scss/details.scss (hunk 2 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done!

File: maquetado/scss/details.scss [hunk 5]
```diff
@@ -1,59 +1,60 @@
...
+      justify-self: center;
     }
-  }
 
   .book-cover {
+    grid-column: 1;
+    grid-row: 3;
     margin-right: 52px;
-    position: relative;
+
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Remove extra line

**Pull request merged**